test/integration/quic_http_integration_test.cc

```diff
@@ -436,5 +443,187 @@ TEST_P(QuicHttpIntegrationTest, ResetRequestWithInvalidCharacter) {
   ASSERT_TRUE(response->waitForReset());
 }
 
+class QuicInplaceLdsIntegrationTest : public QuicHttpIntegrationTest {
```

Comment:
Can we not reuse LdsInplaceUpdateHttpIntegrationTest.ReloadConfigAddingFilterChain and friends?